### DISCUSSION TITLE
About Me: Some Experiences

### DISCUSSION THREAD

**Discussion by ocoke** [ocoke/Biscus#6] [Archived]
I've archived all my blog articles before and started a new story here. 
`[desc_end]`
 ## Coding

I am **Jun**, the author of this blog. I started blogging three years ago. I had to stay at home at that time because of the covid. I was bored and wanted to try something new. I had learned Python before, and I accidentally got into the web because I was trying web crawlers and had to know some basic information about HTML.

Then, I created some small projects, such as the starting page of browsers. Some of them can even visit now (but I don't want to). I started learning about JavaScript and CSS, I realized that only HTML is not enough for building a modern website. I copied lots of codes from open-source projects and other articles to my project. It worked, but not working well. 😂 (like putting all the `<script>` tags inside the `<head>`) .

After that, I wanted other people to visit my website easily so I bought some domains. I felt very cool although the domain name at that time was meaningless (they are all expired now). Because of buying these domains, I started using some modern frameworks for my website: Hexo. Hexo is a static website generator, it has lots of themes and plugins. I changed lots of themes, tried to add all the plugins to my blog to make that "powerful" and it became very slow. And then after that, to make my blog unique, I started writing my own theme.

I put all the "beautiful" elements together and found that the combination of them is so ugly, the page became so dirty. I tried to let them be unified to make sure the page was harmonious. I don't know how I enhance my aesthetics, maybe after visiting a lot of different blogs.

Some people say the performance of a blog is really important because no one wants to wait 10 seconds to open your blog. Well, I started making my blog "lite" and "smooth". This also took me plenty of time. 🤔I explored Service Worker and other tools this time. I added them to my blog, followed their instructions. At the same time, I also care about the accessibility of my blog in China Mainland (I don't have an ICP license 🤣)

I built the page you guys visit now several months ago (March, 2023). About two weeks ago, I started putting my new articles on GitHub Discussions so I don't need to worry about the storage of articles now.

## Life

During these years, I became a high school student, my mind changed a lot, I went to different places in the world, and saw different places and different people.

I love photography and I take lots of photos when I visit a new place. I am planning to put them on social media, maybe later.

By the way, my 16 personalities test is `INFP-T`.

This is my story, now I have started a new blog, nice to meet you again.

## Future

I think I will archive all of my articles in my old blog, they will be able to visit, but they won't appear in the stream. The blog is still under development, I use my hobbies to write these articles, with love. I know a person can't be described by some hashtags, but these are some of my tags:

`#Developer`, `#Photographer`, `#Designer`, `#Coding`, `#Designing`, `#Chinese`, `#INFP`, `#Student`.

**Comment by kikomans3ple** [comment]
https://addrom.com/bypass/

**Comment by IanPP2** [comment]
@kikomans3ple hey
